File: training/to_be_sorted/Mettre en œuvre les design patterns dans vos applications (Orsys: DES)/exercices/code.refactoring/dotnet/cpp/Rental.cpp
#include "StdAfx.h"
#include "Rental.h"


Rental::Rental(Movie^ movie, int daysRented)
        {
            _movie = movie;
            _daysRented = daysRented;
        }
int Rental::GetDaysRented()
        {
            return _daysRented;
        }
Movie^ Rental::GetMovie()
        {
            return _movie;
        }

File: training/to_be_sorted/mettre_en_œuvre_les_design_patterns_dans_vos_applications_orsys_des/exercices/code.refactoring/dotnet/refactoring/Rental.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace csharp
{
    class Rental
    {

        private Movie _movie;
        private int _daysRented;

        public Rental(Movie movie, int daysRented)
        {
            _movie = movie;
            _daysRented = daysRented;
        }
        public int GetDaysRented()
        {
            return _daysRented;
        }
        public Movie GetMovie()
        {
            return _movie;
        }
    }
}
